onyx-remote-driver/src/com/onyx/persistence/manager/impl/DefaultSocketPersistenceManager.java

```diff
@@ -89,7 +87,7 @@ public void saveEntities(List<? extends IManagedEntity> entities) throws EntityE
         try {
             proxyPersistenceManager.saveEntities(entities);
         } catch (RemoteException e) {
-            throw new EntityException(e);
```

Comment:
This is wrong, it should not be the constructor, it should be e.getCause().  That was the root of the problem with some of the unit test failing.  We should not need to instantiate anything, it should be available within the RemoteException